lib/3scale_toolbox/commands/3scale_command.rb

```diff
@@ -12,6 +12,9 @@ def self.command
           usage       '3scale <command> [options]'
           summary     '3scale CLI Toolbox'
           description '3scale CLI tools to manage your API from the terminal.'
+          required :c, 'config-file', '3scale CLI configuration file' do |val, _cmd|
```

Comment:
Passing config file is required?

Reply:
no. When `--config-file` is in the command line, then some value is required.

Reply:
Yep, I found it in the CRI docs. Quite poor naming decision.

Reply:
:+1:  will update definition of commands for clearness